Libs/MRML/Widgets/qMRMLSliceControllerWidget.cxx

```diff
@@ -1548,6 +1567,48 @@ void qMRMLSliceControllerWidgetPrivate::setupRulerMenu()
   rulerMenu->addActions(rulerColorActions->actions());
 }
 
+// --------------------------------------------------------------------------
+void qMRMLSliceControllerWidgetPrivate::setupSlabReconstructionMenu()
+{
+  Q_Q(qMRMLSliceControllerWidget);// Menu
+  this->SlabReconstructionMenu = new QMenu(tr("Slab Reconstruction"), this->ShowSlabReconstructionButton);
+  this->SlabReconstructionMenu->addAction(this->actionShow_slab_reconstruction_widget);
+  this->SlabReconstructionMenu->setObjectName("slabMenu");
+
+  // Slab Reconstruction Thickness
+  QMenu* slabReconstructionThickness = new QMenu(tr("Slab Thickness (mm)"), this->ShowSlabReconstructionButton);
+  slabReconstructionThickness->setObjectName("slicerSpacingManualMode");
+  this->SlabReconstructionThicknessSpinBox = new ctkDoubleSpinBox(slabReconstructionThickness);
+  this->SliceSpacingSpinBox->setDecimals(3);
+  this->SlabReconstructionThicknessSpinBox->setRange(1., VTK_FLOAT_MAX);
+  this->SlabReconstructionThicknessSpinBox->setSingleStep(0.1);
+  this->SlabReconstructionThicknessSpinBox->setValue(1.);
+  QObject::connect(this->SlabReconstructionThicknessSpinBox, SIGNAL(valueChanged(double)),
```

Comment:
Look into adding a function like the following:

```
qMRMLSliderWidget::updateWidgetFromUnitNode
```

Since the current set of change is function, consider doing so in a follow-up commit titled `ENH: Make Thick Slab Reconstruction unit aware`